lib_eio/path.ml

```diff
@@ -35,6 +35,27 @@ let mkdir ~perm ((t:#Fs.dir), path) =
     let bt = Printexc.get_raw_backtrace () in
     Exn.reraise_with_context ex bt "creating directory %a" pp (t, path)
 
+let mkdirs ?(exists_ok=false) ~perm ((t:#Fs.dir), path) =
+  let separate = String.split_on_char (Filename.dir_sep.[0]) in
```

Comment:
I think we decided in #124 that the path separator is always `/` in Eio paths.

Reply:
Oh right, thanks for pointing that out :))